plugins/hook-injector/hook-injector.go

```diff
@@ -30,7 +30,7 @@ import (
 
 	"github.com/containerd/nri/pkg/api"
 	"github.com/containerd/nri/pkg/stub"
-	"github.com/containers/podman/v3/pkg/hooks"
+	hooks "github.com/containers/common/pkg/hooks/1.0.0"
```

Comment:
AFAICT this does not compile if imported like this. I think it should be without the final 'tag', simply as

```
        hooks "github.com/containers/common/pkg/hooks"
```

We're mimicking [cri-o](https://github.com/cri-o/cri-o/blob/2faaecdaf9e3fbbcfe5f640774b610218f6175eb/internal/lib/container_server.go#L11) here.

Reply:
Ok done that.